pkgs/by-name/dw/dwlmsg/README.md

```diff
@@ -0,0 +1,16 @@
+This package depends on DWL being built with the IPC patch found at https://codeberg.org/dwl/dwl-patches/raw/commit/d235f0f88ed069eca234da5a544fb1c6e19f1d33/patches/ipc/ipc.patch
+
+this requires the following to be included in your configuration.nix (or flake)
+
+```
+environment.systemPackages = with pkgs; [
+  (dwl.overrideAttrs (
+    attrs: {
+    patches = [ (fetchpatch {
+        url = "https://codeberg.org/dwl/dwl-patches/raw/commit/d235f0f88ed069eca234da5a544fb1c6e19f1d33/patches/ipc/ipc.patch";
+        sha256 = "sha256-c5225c483aa6217700d2649dd7f67478e2f5f0b60610cc3d0cc0ebedd8356fdf";
+      })
+      ];
+      }))
+    ];
+```
```

Comment:
Can we make this a separate package (say, `dwl-ipc-unstable`)? I'm not a huge fan of this (see https://dont-ship.it/) but if it's clearly marked as unstable not recommended for use, it's probably okay-ish, and less bad than shipping another package that depends on something that doesn't exist in nixpkgs.

Reply:
_I'll repost my responses here & remove them from below as email responses don't work properly._

There is already another package in the repo that relies on the same patch being in place "dwlb"

Perhaps we should raise an issue on the dwl package to include this patch as an option, since it's on their todo list anyway. 

The don't ship it link doesn't really apply here as DWL & subsequent related packages are based on the suckless philosophy, where applying patches is just part of the configuration.

Thankfully nix provides us a mechanism for doing this & still maintaining reproducible builds.

Unfortunately it does require users of these packages to understand how to apply patches via overrideAttrs.

Refer to https://codeberg.org/dwl/dwl-patches